### PR TITLE
Added support for another way of  execution of netsh 

### PR DESCRIPTION
**Pull request by swachchhanda000**
### **Summary of the Pull Request:**
The pull request slightly modifies the existing rule to detect another form of netsh execution

### **Detailed Description of the Pull Request / Additional comments:**
The existing rule can detect the command "netsh interface portproxy add v4tov4 listenaddress=127.0.0.1 listenport=9000 connectaddress=192.168.0.10 connectport=80" but maynot detect if same command is executed through this abbreviated syntax "netsh I p a v l=8001 listena=127.0.0.1 connectp=80 c=192.168.0.11"

### **Example Log Event (In Case of FP Fixes)**
NA

### **Relevant Issues (In Case of Issue Fixes)**
NA

### CODE REVIEW

**Comment by nasbench** [comment]
Hi,

I believe before creating this PR you got a template saying something like this

image

Please consider filling it so that I can review your PR in a timely manner.

Also use `#`to indicate example comments.

And use spaces between the flags to increase atom size

File: rules/windows/process_creation/proc_creation_win_netsh_port_forwarding.yml [hunk 1]
```diff
@@ -22,12 +22,18 @@ detection:
...
             - 'interface'
             - 'portproxy'
             - 'add'
             - 'v4tov4'
     selection_cli_2:
+        CommandLine|contains|all:
+            - 'i'               
+            - 'p'
+            - 'a'
+            - 'v'
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_netsh_port_forwarding.yml:35]:
```suggestion
        CommandLine|contains|all:
            # Example: netsh I p a v l=8001 listena=127.0.0.1 connectp=80 c=192.168.1.1
            - 'i ' # interface               
            - 'p ' # portproxy
            - 'a ' # add
            - 'v ' # v4tov4
```

File: rules/windows/process_creation/proc_creation_win_netsh_port_forwarding.yml [hunk 2]
```diff
@@ -6,9 +6,9 @@ references:
     - https://www.fireeye.com/blog/threat-research/2019/01/bypassing-network-restrictions-through-rdp-tunneling.html
     - https://adepts.of0x.cc/netsh-portproxy-code/
     - https://www.dfirnotes.net/portproxy_detection/
-author: Florian Roth (Nextron Systems), omkar72, oscd.community
+author: Florian Roth (Nextron Systems), omkar72, oscd.community, Swachchhanda Shrawan Poudel
 date: 2019/01/29
-modified: 2023/02/13
+modified: 2023/03/29
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_netsh_port_forwarding.yml:11]:
```suggestion
modified: 2023/04/05
```

**Review by nasbench — CHANGES_REQUESTED** [review]
Can you also please disable the "protected branch" so I can apply suggestions more easily?

**Review by nasbench — APPROVED** [review]

**Comment by nasbench** [comment]
cc @phantinuss fyi

File: rules/windows/process_creation/proc_creation_win_netsh_port_forwarding.yml [hunk 3]
```diff
@@ -6,9 +6,9 @@ references:
     - https://www.fireeye.com/blog/threat-research/2019/01/bypassing-network-restrictions-through-rdp-tunneling.html
     - https://adepts.of0x.cc/netsh-portproxy-code/
     - https://www.dfirnotes.net/portproxy_detection/
-author: Florian Roth (Nextron Systems), omkar72, oscd.community
+author: Florian Roth (Nextron Systems), omkar72, oscd.community, Swachchhanda Shrawan Poudel
 date: 2019/01/29
-modified: 2023/02/13
+modified: 2023/04/05
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_netsh_port_forwarding.yml:11]:
```suggestion
modified: 2023/04/20
```

**Review by phantinuss — APPROVED** [review]

**Pull request merged**